test/parallel/test-stringbytes-external.js

```diff
@@ -129,31 +129,31 @@ var PRE_3OF4_APEX = Math.ceil((EXTERN_APEX / 4) * 3) - RADIOS;
 
   assert.throws(function() {
     buf1.toString();
-  }, /toString failed|Invalid array buffer length/);
+  }, /toString\(\) failed: buffer larger|Invalid array buffer length/);
```

Comment:
Made that fix and tested locally, all is fine.

Not going to force push until CI is done